.github/workflows/pr-downstream.yml

```diff
@@ -57,10 +52,11 @@ jobs:
           annotations-prefix: ${{ runner.os }}-${{ matrix.java-version }}/${{ matrix.maven-version }}
           starting-project: kiegroup/${{ matrix.repository }}
           github-token: "${{ secrets.GITHUB_TOKEN }}"
+          BUILD_MVN_OPTS: ${{ matrix.env_BUILD_MVN_OPTS }}
```

Comment:
maybe, but just wild guess from below lines with the same input

```suggestion
          env:
            BUILD_MVN_OPTS: ${{ matrix.env_BUILD_MVN_OPTS }}
```

Reply:
But in 9.103 as well we did same - https://github.com/kiegroup/kogito-apps/blob/9.103.x-prod/.github/workflows/pr-downstream.yml#L55